likecoin/chain.json

```diff
@@ -24,12 +24,12 @@
     },
     "codebase": {
         "git_repo": "https://github.com/likecoin/likecoin-chain",
-        "recommended_version": "v2.0.0",
+        "recommended_version": "v2.0.2",
         "compatible_versions": [
-            "v2.0.0"
+            "v2.0.2"
         ],
         "binaries": {
-            "linux/amd64": "https://github.com/likecoin/likecoin-chain/releases/download/v2.0.0/likecoin-chain_2.0.0_Linux_x86_64.tar.gz"
+            "linux/amd64": "https://github.com/likecoin/likecoin-chain/releases/download/v2.0.2/likecoin-chain_2.0.2_Linux_x86_64.tar.gz"
```

Comment:
Since you have binaries available for multiple target platforms, it would be good to list them all here rather than just the linux/amd64 one.

Reply:
> Since you have binaries available for multiple target platforms, it would be good to list them all here rather than just the linux/amd64 one.

Updated and rebased